samples/python/aruco_detect_board_charuco.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python
+
+"""aruco_detect_board_charuco.py
+Usage example:
+python aruco_detect_board_charuco.py -w=5 -h=7 -sl=0.04 -ml=0.02 -d=10 -c=tutorial_camera_charuco.yml -i=choriginal.jpg
```

Comment:
The argument parser defaults does not match the proposed example.
Path to image is not correct
The camera calibration file is not provided.